help_to_heat/portal/download_views.py

```diff
@@ -181,22 +183,16 @@ def download_feedback_view(request):
     return response
 
 
-@require_http_methods(["GET"])
-@decorators.requires_service_manager
-def download_referrals_all_view(request):
-    referrals = portal_models.Referral.objects.all().order_by("referral_id")
-    downloaded_at = timezone.now()
-    file_name = downloaded_at.strftime("%d-%m-%Y %H_%M")
-    response = create_referral_xlsx(referrals, file_name, exclude_pii=True)
-    return response
-
-
 @require_http_methods(["GET"])
 @decorators.requires_service_manager
 def download_referrals_last_week_view(request):
-    referrals = portal_models.Referral.objects.filter(created_at__gte=datetime.now() - timedelta(days=7)).order_by("referral_id")
-    downloaded_at = timezone.now()
-    file_name = downloaded_at.strftime("%d-%m-%Y %H_%M")
+    today = date.today()
+    end_of_last_week = today - timedelta(days=today.weekday())
+    start_of_last_week = end_of_last_week - timedelta(days=7)
```

Comment:
I think we can actually use:
```
- timedelta(weeks=1)
```
https://docs.python.org/3/library/datetime.html#timedelta-objects

Reply:
Changed this! Is the advantage here increased readability?

Reply:
Yes, just a style thing. We may as well use it, since it's available.